manifest.json

```diff
@@ -1,12 +1,19 @@
 {
   "id": "com.draculatheme.dracula-replugged",
   "name": "Dracula",
-  "description": "Dark theme for Replugged ",
-  "author": {
-    "name": "Dracula",
-    "discordID": "971847378011308102",
-    "github": "dracula"
-  },
+  "description": "Dark theme for Replugged",
+  "author": [
+    {
```

Comment:
i am (evidently) no longer maintaining the theme css. if you could remove me from the authors section i'll merge this.